web/packages/core/src/internal/ui/panic.tsx

```diff
@@ -154,14 +155,29 @@ function createPanicError(error: Error | null): {
     actions: PanicAction[];
 } {
     if (error instanceof LoadSwfError) {
-        if (error.swfUrl && !error.swfUrl.protocol.includes("http")) {
-            // Loading a swf on the `file:` protocol
+        const urlExtensionProtocol = isExtension ? document.baseURI.split(":")[0] + ":" : "";
+        if (error.swfUrl && error.swfUrl.protocol === urlExtensionProtocol) {
+            // The user entered an invalid URL
+            const urlExtensionPart = document.baseURI.split("player.html")[0]!;
             return {
-                body: textAsParagraphs("error-file-protocol"),
-                actions: [
-                    CommonActions.OpenDemo,
-                    CommonActions.DownloadDesktop,
-                ],
+                body: textAsParagraphs("error-no-valid-url", {
+                    url: error.swfUrl.href.replace(urlExtensionPart, ""),
+                }),
+                actions: [CommonActions.ShowDetails],
+            };
```

Comment:
Thanks for noticing this! I fixed it so that all these extension player - specific error messages (as well as the http advice) are only shown in the extension player and a new additional error message is displayed if a website is misconfigured and uses a different protocol.